src/components/list-lemplate/actions-menu.tsx

```diff
@@ -40,11 +40,11 @@ const ActionsMenu = ({ onUpdate, onDelete, pageName, record }: ActionProps) => {
 
     return (
         <Dropdown menu={{ items }} trigger={["click"]}>
-            <a onClick={(e) => e.preventDefault()}>
+            <Button>
```

Comment:
```suggestion
            <Button type={"text"}>
```

it's better to add type="text" to this scope of buttons if we want them to not be outlined